### ISSUE TITLE
Selected/Unselected icon color in tablet mode not consistent

### ISSUE THREAD

**Issue by D-Jeremie** [rodydavis/navigation_rail#4]
First of all, thank you for your great widget ! 

I just noticed that in Tablet mode the color of selected and unselected icons are not consistent with phone mode. 
Maybe it's a feature that you choose ?

If not you could consider this :

Inside _buildTab you could easily make it  consistent with `final _color = selected
        ? bottomNavigationBarSelectedColor : bottomNavigationBarUnselectedColor;` 
instead of `final _color = selected
        ? _isDark ? Colors.tealAccent[200] : _theme.primaryColor
        : Colors.grey;`

**Comment by rodydavis** [comment]
Yep just saw this. I fixed it on my local copy. I will push a fix soon

**Comment by rodydavis** [comment]
Fixed on dev version: https://pub.dev/packages/navigation_rail/versions/1.6.0-dev.1

Can you test?
